plugins/slack/app/models/slack_notification.rb

```diff
@@ -4,18 +4,20 @@ def initialize(deploy)
     @stage = deploy.stage
     @project = @stage.project
     @user = @deploy.user
+    @webhook = @stage.slack_webhooks.first
```

Comment:
You'll only end up notifying the first webhook, you should iterate inside #deliver